### DISCUSSION TITLE
How to stand up a separate ChromaDB service and connect that to Vanna?

### DISCUSSION THREAD

**Discussion by MichaelMMeskhi** [vanna-ai/vanna#625] [Q&A] [answered]
I am looking to have a stand alone docker image of ChromaDB running, is there a way to direct the connection to vanna?

**Comment by zainhoda** [comment, marked as answer]
Yes, you can pass in a `chromadb.api.client.Client` as a `client` parameter to the `config`:

https://github.com/vanna-ai/vanna/blob/main/src/vanna/chromadb/chromadb_vector.py#L37
